src/components/map/MapFilter.js

```diff
@@ -45,9 +45,7 @@ const MapFilter = () => {
   return (
     <StyledDiv>
       <Map filteredData={searchResults} />
-
-      {/* Commenting this out because I will move filter logic (Essien) */}
-      {/* <StyledFilterDiv>
+      <StyledFilterDiv>
```

Comment:
@khwanchaiwill Any particular reason the filter code here was uncommented ? I commented it out because may be moving it as the filter/search component was moved into the nav overlaying the map/chart. cC @mtruj013 @npm-nic @jlong5795